app/python_dhcp_server

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+cd /app/config
+
+if test -f /app/server/dhcpgui.conf.bak ; then
+  # During creation of the flatpak, we made a backup of the configuration and a link.
+  echo "I will try to allow the user to edit the configuration."
+  if ! test -f dhcpgui.conf ; then
+    echo "No configuration detected. Creating a new one."
+    ls -la /app/server/ /app/config/
```

Comment:
`/app` is read only. I doubt you even tested this.

Reply:
/data also seems read-only. I wonder where to put the data at all so that it can be stored in between runs of the program.

Reply:
it should be following the xdg directory spec

https://specifications.freedesktop.org/basedir-spec/latest/ar01s02.html

as mentionned there

https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories

Reply:
Thanks, I use the right directories now and the state persists. This is commenting at outdated code.